README.md

```diff
@@ -33,6 +33,7 @@ List of resources and project samples for Engineering Students
   - [MySQL <a name="mysql"></a>](#mysql-)
   - [Java Programming <a name="java-programming"></a>](#java-programming-)
   - [Kotlin Programming <a name="kotlin-programming"></a>](#kotlin-programming-)
+  - [Jetpack Componse <a name="jetpack-compose"></a>](#jetpack-compose)
```

Comment:
Please correct the spelling

Reply:
Also, the link is not pointing to the section as well

Reply:
Hello, completed
### Fix
- Typo in README
- Links to source
-
Thank You.